tests/ui/perfherder/alerts-view/status_dropdown_test.jsx

```diff
@@ -85,3 +89,231 @@ test("Tags modal opens from 'Edit tags'", async () => {
 
   expect(modal).toBeInTheDocument();
 });
+
+test('Triage to countdown shows 3 working days left', async () => {
+  const alert = testAlertSummaries[0];
+  alert.created = '2022-02-07T11:41:31.419156';
+
+  Date.now = jest.fn(() => Date.parse('2022-02-7'));
```

Comment:
Here as well a comment with "Setting the current date to Monday (same day the alert was created)." would be nice.